projects/packen-ng/README.md

```diff
@@ -690,6 +690,8 @@ A special dropdown with more details to show.
 - **labelPosition**: Defines label position `(top|bottom)` **NOTE**: default value is bottom.
 - **icon**: Defines icon to show side the name.
 - **required**: Defines if the component is required `(false|true)`.
+- **phantom**: Defines if background will be transparent or not.`(true|false)`.
+- **disabled**: Defines if component are enabled or not: `(true|false)`.
```

Comment:
If component is enabled.

Reply:
En los inputs se usa de este modo (disabled="true") lo que quiere decir que no esta habilitado, por lo tanto decidimos manejarlo de este modo.